exercises/strain/strain_test.go

```diff
@@ -19,6 +19,14 @@ import (
 	"testing"
 )
 
+const targetTestVersion = 1
+
+func TestTestVersion(t *testing.T) {
```

Comment:
In some exercises, I realized we put it as a first test but if there are helper functions, they go first. 
So is it supposed to be the first thing in the file or the first test when tests started ? 

Otherwise, fine.